### PR TITLE
change fixit to docsprint; add list of project 'minimum viable documentation'

### PR DESCRIPTION
**Pull request by bsweger**
Two things happening here:
1. Change fixit to doc sprint
2. Add the list of 'minimum viable documentation' required for projects

### CODE REVIEW

File: pages/docsprint/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Project teams and working groups get their onboarding houses in order. At a minimum, the documentation for each project should contain the following information:
+
+#### In Project Repository
+
+Make sure README.md has:
+
+* A link to an existing public instance of the project. Links to internal/staging versions can be listed on the Hub (see below) if you don't want them publicly visible.
+* Description of the workflow for submitting project code changes (_e.g._ do you fork or branch?)
+* How to stand up a development version of the project. Installs, scripts, a Vagrantfile for forced fresh install, etc.
```
Comment (mbland) [inline pages/docsprint/onboarding.md]: I'd replace "Installs", with "Installment commands"

File: pages/docsprint/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* Project management-related information:
+
+    * links to sprint boards (some are in GitHub, some in Trello, etc.)
+    * links to task management tools (GitHub, Trello, Pivotal)
```
Comment (mbland) [inline pages/docsprint/onboarding.md]: Pivotal Tracker

File: pages/docsprint/onboarding.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    * Quick write-up of where things are: GitHub issues versus sprint plans
```
Comment (mbland) [inline pages/docsprint/onboarding.md]: "GitHub issues, sprint plans, dashboards, etc."

File: pages/docsprint/onboarding.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Update the CONTRIBUTING file to reflect the workflow as written in README.md.
```
Comment (mbland) [inline pages/docsprint/onboarding.md]: Before this, I'd add: `Also consider:` and making the rest a bulleted list, i.e.:
```markdown
* Updating the `CONTRIBUTING` file...
* Tagging "starter project" GitHub issues as `new-to-project`
```

File: pages/docsprint/onboarding.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Tag issues as new-to-project.
+
+#### On the Hub
+
+Make sure your project has a Hub page and that it contains the following information:
```
Comment (mbland) [inline pages/docsprint/onboarding.md]: I'm thinking that for every project in `_data/private/projects`, I may auto-create a page in `pages/private` a la https://hub.18f.us/private/18f-site/. Then we can mention that project folks should be able to use that area to add/maintain some of this information.

**Comment by bsweger** [comment]
Thanks for the edits. Made the suggested changes with the exception of the project pages, which sounds like it needs to be fleshed out a bit.

**Pull request merged**